test/helpers/kubectl.go

```diff
@@ -288,32 +291,32 @@ func (kub *Kubectl) CiliumEndpointPolicyVersion(pod string) map[string]int64 {
 
 //CiliumExec runs cmd in the specified Cilium pod
 func (kub *Kubectl) CiliumExec(pod string, cmd string) *CmdRes {
-	cmd = fmt.Sprintf("kubectl exec -n kube-system %s -- %s", pod, cmd)
+	cmd = fmt.Sprintf("%s exec -n kube-system %s -- %s", kubectl, pod, cmd)
 	return kub.Node.Exec(cmd)
 }
 
 //CiliumNodesWait When a new cilium DS is set, the status can be ok but tunnels
```

Comment:
Please add spaces after slashes.

Reply:
There needs to be a considerable amount of cleanup in the documentation of this code. To avoid having this PR balloon, I've created another issue to resolve these formatting and documentation issues: #2084. I went through and fixed the spacing and grammatical mistakes in `helpers/kubectl.go` and `helpers/cilium.go` which I will upload shortly. Is this OK?

Reply:
Done.